### PR TITLE
[RW-654][risk=no] Show CDR name instead of version

### PR DESCRIPTION
**Pull request by calbach**
Demo: https://calbach-dot-all-of-us-workbench-test.appspot.com/workspaces

Note: there may be a brief delay before the name is displayed - during this time it will appear blank.

image

### CODE REVIEW

File: ui/src/app/views/workspace/component.spec.ts [hunk 1]
```diff
@@ -149,26 +115,37 @@ describe('WorkspaceComponent', () => {
...
         }
       ]}).compileComponents().then(() => {
-        workspacePage = new WorkspacePage(TestBed);
+        fixture = TestBed.createComponent(WorkspaceComponent);
+        updateAndTick(fixture);
       });
-      tick();
   }));
 
   it('displays correct notebook information', fakeAsync(() => {
```
Comment (blrubenstein) [inline ui/src/app/views/workspace/component.spec.ts]: Is this test old? Are we still displaying the notebooks on the workspace about page?

Comment (calbach) [reply ui/src/app/views/workspace/component.spec.ts]: It's very old. We are still loading notebooks here though so we can put them in the scroll component.

This test is pretty useless though, I'm just going to drop this test case and leave only the research purpose check as a basic render verification.

**Comment by calbach** [comment]
I'll deploy a demo and check with @lstrano-broad on this.

**Comment by calbach** [comment]
@lstrano-broad please check out the "about workspace" page on my demo server: https://calbach-dot-all-of-us-workbench-test.appspot.com/workspaces

**Comment by calbach** [comment]
Per offline discussion with Lou and Karthik, just going with a simple "Loading..." text for now. We think a spinner here would probably be too distracting, and it loads in fairly quickly anyways.

**Pull request merged**